pom.xml

```diff
@@ -288,28 +287,29 @@
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<version>${jacoco-maven-plugin.version}</version>
+				<configuration>
+					<append>true</append>
+					<destFile>${sonar.jacoco.reports}/jacoco.exec</destFile>
+					<dataFile>${sonar.jacoco.reports}/jacoco.exec</dataFile>
+					<outputDirectory>${sonar.jacoco.reports}/jacoco</outputDirectory>
+				</configuration>
+
 				<executions>
 					<execution>
 						<id>pre-unit-test</id>
 						<goals>
 							<goal>prepare-agent</goal>
 						</goals>
 						<configuration>
-							<!-- Sets the path to the file which contains the execution model. -->
-							<destFile>${sonar.jacoco.reports}/jacoco.exec</destFile>
 							<propertyName>surefireArgLine</propertyName>
```

Comment:
In other POMs this uses placeholder syntax like
```
<propertyName>surefireArgLine</propertyName>
```
Can you check which is correct?